### DISCUSSION TITLE
How to set Backgroundcolor of images

### DISCUSSION THREAD

**Discussion by UkeHa** [dotnet/maui#16363] [Q&A]
I'm trying to change the background of an image so it fits the app theme but both background as well as backgroundcolor don't affect anything once the image itself has been loaded. Neither does coloring the contentframe.  Has anyone a clue as to how to do it, if it's possible at all?

```
<controls:ContentFrame Grid.Row="0"
                       Padding="0" 
                       IsVisible="{Binding ProductImageVisible}">
		      <Image
                       Source="{Binding ProductImageSource}"
                       IsVisible="{Binding ProductImageVisible}"/>
				</controls:ContentFrame>
```

![grafik](https://github.com/dotnet/maui/assets/2254304/bd7faaf8-3619-4ea1-8300-05d600596c82)

**Comment by JonnySKK** [comment]
Have you tried wrapping the image in a frame or a border instead? Background colour should work on those controls. Is the ContentFrame a custom control you have created?

**Reply by UkeHa** [reply]
using a frame in a grid doesn't work either.

<Frame Grid.Row="0" BackgroundColor="Red">
					<Image
						BackgroundColor="Green"
                       Source="{Binding ProductImageSource}"
                       IsVisible="{Binding ProductImageVisible}"/>
					</Frame>

![grafik](https://github.com/dotnet/maui/assets/2254304/6f242f84-e775-4458-90a2-6d89188711e5)

Neither does border:

![grafik](https://github.com/dotnet/maui/assets/2254304/6b2b7136-bcdc-4b39-9b17-990490b814bb)
